pkg/install/util.go

```diff
@@ -37,6 +41,49 @@ func GetPort(cn *corev1.Container, portName string) (*corev1.ContainerPort, erro
 	return nil, fmt.Errorf("unable to locate port %q in container %q", portName, cn.Name)
 }
 
+// GetAppProto determines the application protocol of the given ServicePort. The given AppProtocolStrategy
+// used if the port's appProtocol attribute is unset.
+func GetAppProto(ctx context.Context, aps client.AppProtocolStrategy, p *corev1.ServicePort) string {
```

Comment:
I'm just gonna dump the link to the KEP [here](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1507-app-protocol) because i did not know about `appProtocol`